src/main/resources/anniv6Resources/localization/eng/BrokenSpace/Relicstrings.json

```diff
@@ -0,0 +1,105 @@
+{
+  "${ModID}:BrokenDataDisk": {
+    "NAME": "Broken Data Disk",
+    "FLAVOR": "no",
+    "DESCRIPTIONS": [
+      "At the start of each combat, lose #b",
+      " #yFocus. The first time you channel an orb each turn, channel another orb of the same type."
+    ]
+  },
+  "${ModID}:BrokenSneckoSkull": {
+    "NAME": "Broken Snecko Skull",
+    "FLAVOR": "no",
+    "DESCRIPTIONS": [
+      "When you gain a buff, gain #b",
+      " more."
+    ]
+  },
+  "${ModID}:BrokenRedSkull": {
+    "NAME": "Broken Red Skull",
+    "FLAVOR": "no",
+    "DESCRIPTIONS": [
+      "When you are above #b",
+      "% #yHP, you have #b",
+      " extra #yDexterity."
+    ]
+  },
+  "${ModID}:BrokenOmamori": {
+    "NAME": "Broken Omamori",
+    "FLAVOR": "no",
+    "DESCRIPTIONS": [
+      "Add #b",
+      " #yCurses to your deck. Whenever you rest, upgrade #b",
+      " random cards in your deck."
+    ]
+  },
+  "${ModID}:BrokenDamaru": {
+    "NAME": "Broken Damaru",
+    "FLAVOR": "no",
+    "DESCRIPTIONS": [
+      "At the end of your turn, you may #yExhaust a card in your hand. Gain #yMantra equal to it's cost.",
+      "Select a card to exhaust for mantra."
+
+    ]
+  },
+  "${ModID}:BrokenWingBoots": {
+    "NAME": "Broken Wing Boots",
+    "FLAVOR": "no",
+    "DESCRIPTIONS": [
+      "When you would enter an event in a ? room, choose between two events."
+
+    ]
+  },
+  "${ModID}:BrokenSozu": {
+    "NAME": "Broken Sozu",
+    "FLAVOR": "no",
+    "DESCRIPTIONS": [
+      "In-Combat Potions cost #b",
```

Comment:
Interesting effect. It seems reasonably balanced, though if it turns out to be too weak you could consider having it lose energy next turn instead of on the current turn. That would be a bit more forgiving to the player.

Reply:
For the text of this relic, seeing "1 [E]" is weird. Normally you don't have the number when it's a fixed small amount, so this should just be "In-Combat potions cost [E] to use."